tooling/template/next.config.mjs

```diff
@@ -9,6 +64,27 @@ const nextConfig = {
   turbo: {
     moduleIdStrategy: "deterministic",
   },
+  async headers() {
```

Comment:
issue: I don't think this would work, as it is mentioned by NextJS that [headers is unsupported for static exports](https://nextjs.org/docs/pages/building-your-application/deploying/static-exports#unsupported-features). The headers should be applied on the CloudFront distribution directly instead via Behaviors > Response headers policy.

Right now we are using the managed policy provided by AWS, so we will need to transition to our own custom response headers policy, then specify the CSP there.

<img width="1105" alt="image" src="https://github.com/user-attachments/assets/d31fc741-a654-465d-abff-9e40b4623609" />

Reply:
ah thats right! will close this PR